### DISCUSSION TITLE
How is authentication persisted in the API?

### DISCUSSION THREAD

**Discussion by wsw70** [usememos/memos#3801] [Q&A] [answered]
The [documentation for the API](https://memos.apidocumentation.com/reference) explains how to [sign in a user](https://memos.apidocumentation.com/reference#tag/authservice/POST/api/v1/auth/signin).

The response however does not mention ant token to be reused in further exchanges with the API. In that case how is the authentication persisted between API calls?

Or in other words: what should I do in my code to authenticate further requests?

**Comment by boojack** [comment, marked as answer]
You can create an access token in user setting page. Reference docs: https://www.usememos.com/docs/security/access-tokens

**Reply by wsw70** [reply]
Yes, I found that in the meantime, thanks. 

In that case what is the authentication via login and password for?
